index.js

```diff
@@ -27,145 +43,199 @@ BroccoliMergeTrees.prototype.debug = function(message, args) {
 }
 
 BroccoliMergeTrees.prototype.build = function() {
-  var inputPaths = this.inputPaths
-  var outputPath = this.outputPath
-  var overwrite = this.options.overwrite
   this._buildCount++;
 
   var start = new Date()
-  mergeRelativePath('')
+
+  var fileInfos = this._mergeRelativePath('');
+  var entries = fileInfos.map(function(fileInfo) {
+    return fileInfo.entry;
+  });
+  var indices = fileInfos.map(function(fileInfo) {
+    return fileInfo.indices;
+  });
```

Comment:
could be, but the difference is still very much "in the noise" when it comes tooverall costs